docs/tutorial/quick-start.md

```diff
@@ -1,201 +1,460 @@
-# Quick Start Guide
+# Quick Start
 
-## Quickstart
+This guide will step you through the process of creating a barebones Hello World app in
+Electron, similar to [`electron/electron-quick-start`][quick-start].
 
-Electron is a framework that enables you to create desktop applications with JavaScript, HTML, and CSS. These applications can then be packaged to run directly on macOS, Windows, or Linux, or distributed via the Mac App Store or the Microsoft Store.
+By the end of this tutorial, your app will open a browser window that displays a web page
+with information about which Chromium, Node.js, and Electron versions are running.
 
-Typically, you create a desktop application for an operating system (OS) using each operating system's specific native application frameworks. Electron makes it possible to write your application once using technologies that you already know.
+[quick-start]: https://github.com/electron/electron-quick-start
 
-### Prerequisites
+## Prerequisites
 
-Before proceeding with Electron you need to install [Node.js][node-download].
-We recommend that you install either the latest `LTS` or `Current` version available.
+To use Electron, you need to install [Node.js][node-download]. We recommend that you
+use the latest `LTS` version available.
 
 > Please install Node.js using pre-built installers for your platform.
 > You may encounter incompatibility issues with different development tools otherwise.
 
-To check that Node.js was installed correctly, type the following commands in your terminal client:
+To check that Node.js was installed correctly, type the following commands in your
+terminal client:
 
 ```sh
 node -v
 npm -v
 ```
 
 The commands should print the versions of Node.js and npm accordingly.
-If both commands succeeded, you are ready to install Electron.
 
-### Create a basic application
+**Note:** Since Electron embeds Node.js into its binary, the version of Node running
+your code is unrelated to the version running on your system.
 
-From a development perspective, an Electron application is essentially a Node.js application. This means that the starting point of your Electron application will be a `package.json` file like in any other Node.js application. A minimal Electron application has the following structure:
+[node-download]: https://nodejs.org/en/download/
+
+## Create your application
+
+### Scaffold the project
+
+Electron apps follow the same general structure as other Node.js projects.
+Start by creating a folder and initializing an npm package.
 
-```plaintext
-my-electron-app/
-├── package.json
-├── main.js
-├── preload.js
-└── index.html
+```sh npm2yarn
```

Comment:
That looks like a nice magical directive...

Reply:
Yep, works out of the box with Docusaurus. Will be a no-op on the current website.